### PR TITLE
chore: Rename variables.

### PR DESCRIPTION
**Pull request by IRONICBo**
In this PR, we fixed the issue #449 to enhance code readability and conciseness.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -69,6 +69,7 @@
     clippy::impl_trait_in_params,  // Allow impl AsRef<Path>, it's common in Rust
     clippy::missing_assert_message, // Allow assert! without message, mainly in test code
     clippy::semicolon_outside_block, // We need to choose between this and `semicolon_inside_block`, we choose outside
+    clippy::similar_names, // Allow similar names
```
Comment (xiaguan) [inline src/main.rs:72]: Provide specific reasons, due to the existence of uid and gid

**Comment by xiaguan** [comment]
1. The priority of using rebase is higher than merge
2. The commit message should be clearer, specifying which variables have been renamed

File: src/main.rs (hunk 1 above)
Comment (IRONICBo) [reply src/main.rs:72]: Ok, I will update it.

**Comment by xiaguan** [comment]
Update the branch and rerun failed tests.

**Pull request closed without merging**